frontend-ssr/components/cli_git/git_commands_cc.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+interface Props {
+  defaultValue:Array<RegExp> | Array<string>
```

Comment:
Let's change the name of this prop. `defaultValue` is not meaningful enough. Let's call it "commands".

Reply:
Also, `eslint` I think it should have complained about the lack of space between `defaultValue:` and `Array<RegExp`.

Reply:
Also also, fyi: you can also do `RegExp[]` instead of `Array<RegExp`. But you don't have to change it, use whichever you like the most.

Reply:
Ok. I'll do RegExp[].